docs/source/documentation/fibertube_tracking.rst

```diff
@@ -199,21 +199,35 @@ Tracking
 
 When the tracking algorithm is about to select a new direction to
 propagate the current streamline, it will build a sphere of radius
-``blur_radius`` and pick randomly from all the fibertube segments
-intersecting with it. The larger the intersection volume, the more
-likely a fibertube segment is to be picked and used as a tracking
+``blur_radius`` and examine all the fibertube segments intersecting
+with it. We get a list of segment directions with each a probability
+of getting picked. This propability is based on the segment's volume of
+intersection with the sphere. So the more a fibertube segment intersects
+the sphere, the more likely it is to be picked and used as a tracking
 direction.
 
-
-.. image:: https://github.com/user-attachments/assets/0308c206-c396-41c5-a0e1-bb69b692c101
+.. image:: https://github.com/user-attachments/assets/782bb6d2-0e5c-48a5-8606-1d95d6675e0d
    :alt: Visualization of the blurring sphere intersecting with segments
 
+This is similar to computing the Track Orientation Distribution (TOD)
+but it is not yet represented as a spherical function. It is merely
+an array of directions. Unfortunately, this process is a little too good
+when compared to normal tracking. This is because each potential direction
```

Comment:
"Unfortunately, ..." -> This does not sound like science. It is not unfortunate. It is explanable and a scientific fact. Please rephrase and keep a boring scientific writing style :)